### PR TITLE
Add Print/PDF Export View

### PR DESCRIPTION
**Pull request by timengel**
## Changelog

- [X] Added compact print view that can be used for PDF exports
- [X] "Show Authors of Notes" propagates to the print view
- [X] Print dialog opens automatically from the start and can be reopened manually via a floating button

Note: Some background colors are missing if "Show backgrounds" is disabled in the print dialog (it was enabled from start in Chrome but not in Firefox).

## ToDo
- [x] Fix tests
- [x] Check note order/sorting without votes
- [x] Rotes Herz
- [x] Rechtsbündiges Datum + Line Height (Participants und Datum mehr Abstand)
- [x] Rahmen um Stacks
- [x] A4 Format, keine Notes abschneiden

## Off Topic
- [x] Fix JSON and CSV export filenames

## Checklist
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] The light- and dark-theme are both supported and tested
- [x] The design was implemented and is responsive for all devices and screen sizes
- [x] The application was tested in the most commonly used browsers (e.g. Chrome, Firefox, Safari)

### CODE REVIEW

**Comment by timengel** [comment]
image

File: src/assets/icon-print.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg width="512px" height="512px" viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg">
```
Comment (CronJorian) [inline src/assets/icon-print.svg:1]:
```suggestion
<svg viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg">
```

Removing width and height will make things much easier.

File: src/components/SettingsDialog/ExportBoard/PrintView/PrintView.scss [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+.print-view__button {
+  background: lighten($color-button, 35%);
+  border: solid 1px lighten($color-button, 35%);
+  border-radius: $border-radius--round;
+  margin: $margin--small;
+  cursor: pointer;
+  font-weight: bold;
+  font-size: 16px;
+  padding: 3px 5px;
```
Comment (CronJorian) [inline src/components/SettingsDialog/ExportBoard/PrintView/PrintView.scss:30]: There are better ways to handle icons than using random values to apply centering.

Due to the `icon-close.svg` having margin, it needs to be larger than the print icon, but it does not require arbitrary padding and margin values.

File: src/components/SettingsDialog/ExportBoard/PrintView/PrintView.tsx [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+  const getAuthorName = (authorId: string) => (boardData?.board.showAuthors ? boardData?.participants.filter((p) => p.user?.id === authorId)[0].user?.name : "");
+
+  const getNoteVotes = (noteId: string) => (boardData?.votings ? boardData?.votings[0].votes?.votesPerNote[noteId]?.total ?? 0 : 0);
+
+  const voteLabel = (noteId: string) => {
+    if (boardData?.votings && boardData?.board.showVoting) {
+      const votes = getNoteVotes(noteId);
+      return votes > 0 ? <div className="print-view__note-info-votes">{votes} Votes</div> : "";
+    }
+    return "";
```
Comment (CronJorian) [inline src/components/SettingsDialog/ExportBoard/PrintView/PrintView.tsx:103]: Could do an early out to increase readability.

**Review by CronJorian — APPROVED** [review]

**Pull request merged**